dsl/docker-multiarch/images/solr.groovy

```diff
@@ -0,0 +1,38 @@
+import vars.multiarch
+
+for (arch in multiarch.allArches()) {
+        meta = multiarch.meta(getClass(), arch)
+
+        freeStyleJob(meta.name) {
+                description(meta.description)
+                logRotator { daysToKeep(30) }
+                label(meta.label)
+                scm {
+                        git {
+                                remote { url('https://github.com/docker-solr/docker-solr') }
+                                branches('*/master')
+                                clean()
+                        }
+                }
+                triggers {
+                        upstream("docker-${arch}-openjdk", 'UNSTABLE')
+                        scm('H H/6 * * *')
+                }
+                wrappers { colorizeOutput() }
+                steps {
+                        shell(multiarch.templateArgs(meta) + '''
+sed -i "s!^FROM    !FROM    $prefix/!" */Dockerfile
+
+latest="$(./generate-stackbrew-library.sh | awk '$1 == "latest:" { print $3; exit }')"
+
+for v in */; do
+        v="${v%/}"
+        docker build -t "$repo:$v" "$v"
+        if [ "$v" = "$latest" ]; then
+                docker tag -f "$repo:$v" "$repo"
```

Comment:
It looks like this whole file needs `s/ {8}/\t/g` :wink:  (converting spaces to tabs)